src/oc/hoc.cpp

```diff
@@ -960,16 +958,10 @@ void hoc_final_exit(void) {
     rl_deprep_terminal();
 #endif
     ivoc_cleanup();
-#ifdef WIN32
-    hoc_win32_cleanup();
-#else
-    std::string cmd{neuron_home};
-    cmd += "/lib/cleanup ";
-    cmd += std::to_string(hoc_pid());
-    if (system(cmd.c_str())) {  // fix warning: ignoring return value
-        return;
-    }
-#endif
+    auto tmp_dir = std::getenv("TEMP");
+    auto path = std::filesystem::path(tmp_dir ? std::string(tmp_dir) : "/tmp") /
+                ("oc" + std::to_string(hoc_pid())) / ".hl";
+    std::filesystem::remove(path);
```

Comment:
This can also take an `std::error_code` in case we want to silently ignore any errors.

Reply:
Should we merge or do you prefer to figure this out before?

Reply:
Since the original runs `rm -f`, I think we should use the variant that ignores the error code, unless we specifically want to change that behavior.